src/util/sqlhelper.cpp

```diff
@@ -237,4 +228,44 @@ namespace hsql {
     }
   }
 
+  std::ostream& operator<<(std::ostream& os, const OperatorType& op) {
+    static const std::map<const OperatorType, const std::string> operatorToToken = {
```

Comment:
maybe use a normal switch here? https://stackoverflow.com/questions/931890/what-is-more-efficient-a-switch-case-or-an-stdmap 

minor

Reply:
Interesting read. I would like to argue that this particular method is not subject to performance concerns. It's a debug helper for printing a parse tree directly to `cout`. Even larger query plans will contain a modest number of operators.
In this case I would favor conciseness and readability over lookup-performance. I think creating this map only once accounts for the performance aspect adequately.

Reply:
As `operator<<` should never be a performance-critical method, I don't think that it makes sense to spend time here.